.changeset/blue-stingrays-change.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Changed: `color-named` message arguments' order
```

Comment:
@jeddy3 @ybiquitous
Should https://github.com/stylelint/stylelint/pull/8239/files#diff-2ca0a46038dd09bf8aada629daad88026083ec81a1600341b092f003e9111985R152 be considered a breaking change?
i.e. users of https://stylelint.io/user-guide/configure/#message